src/UnitTests/test_quote_prov.cpp

```diff
@@ -1,25 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include "unit_test.h"
 #include "dcap_provider.h"
+#include "ext\intel\sgx_ql_lib_common.h"
 #include "local_cache.h"
-#include "UnitTests/unit_test.h"
 
+#if defined(__LINUX__)
+#include <dlfcn.h>
+#include <tgmath.h>
 #include <cstdio>
 #include <cstring>
 #include <ctime>
 #include <memory>
-#include <dlfcn.h>

```

Comment:
Why are you reordering these headers?

Reply:
Careless mistake on my part. noted and reverting it